src/main/webapp/components/table/TableWrapper.jsx

```diff
@@ -270,6 +288,33 @@ function TableWrapper({ page, serviceName, handleRequestModalOpen, handleOpenPag
                 tableConfig={tableConfig}
                 services={services}
             />
+            {displayBtnDisableAllRows && totalElement > 1 ? ( // TODO: find a correct place and design for that element
```

Comment:
I would extract the all these two components with it's logic to separate component. TableWrapper is already big. But it is up to you.

Reply:
moved it to separate component